crates/biome_service/src/file_handlers/javascript.rs

```diff
@@ -659,13 +659,18 @@ pub(crate) fn fix_all(params: FixAllParams) -> Result<FixFileResult, WorkspaceEr
                 }
 
                 for action in signal.actions() {
-                    // suppression actions should not be part of the fixes (safe or suggested)
-                    if action.is_suppression() {
-                        continue;
-                    }
-
                     match params.fix_file_mode {
+                        FixFileMode::ApplySuppressions => {
+                            if action.is_suppression()
+                                && action.applicability == Applicability::Always
```

Comment:
I don't think we need to check the applicability, unless there's some logic I am missing